### DISCUSSION TITLE
Mounting suggestions

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#132] [Q&A] [answered]
Ive got a warco WM180 and its a pretty small lathe but it cuts decently well and until I can convince my wife I need a bigger lathe it will have to do.

The problem is that space is at a premium on this lathe and after getting the encoder it seems that its a lot bigger than what I thought it was going to be and as such it doesnt really fit and line up with the gear for the spindle.

I thought maybe I could flip it around and 3d print a mount that holds it, the issue then is my cover wont fit on.

I wonder if the encoder I got was the wrong size. Whats the normal size for them?

![387415721_878490116539002_8572612914868861774_n](https://github.com/kachurovskiy/nanoels/assets/2408274/fc4a1e1a-852f-4f41-921d-4d67f4c12aa3)
![370323019_6630692417013928_5931577927610713461_n](https://github.com/kachurovskiy/nanoels/assets/2408274/89a0a99b-cd61-47b8-b76d-c848eee44b55)

**Comment by kachurovskiy** [comment, marked as answer]
All encoders I've seen are approx. the same size - 38.3mm in diameter and 35mm tall main body with a 20mm mounting shoulder, 6mm shaft.

One option is to mount encoder as you show and use a gear with a conical head to try and work around the motor pulley:

![image](https://github.com/kachurovskiy/nanoels/assets/517919/6f6f584b-d8e1-4cf2-bfa4-408c9d7c56e6)
[EncoderMount38.4mm.FCStd.zip](https://github.com/kachurovskiy/nanoels/files/12910756/EncoderMount38.4mm.FCStd.zip)

Another option is to turn the encoder around and cut a hole in the lathe cover if necessary:

![image](https://github.com/kachurovskiy/nanoels/assets/517919/41ac4db6-3e8f-438d-b382-a856a21010df)
[EncoderMountAndGear.zip](https://github.com/kachurovskiy/nanoels/files/12910760/EncoderMountAndGear.zip)

**Reply by Festivejelly** [reply]
Hrm I dont think the standard gears will work due to space constrains. Id need something like an 80 tooth gear which would end up poking through the case.

Maybe I could mount it in the cavity and drill a hole through it.

![395089433_1392727184996406_5385939392791492304_n](https://github.com/kachurovskiy/nanoels/assets/2408274/04493e74-58cc-42b1-95ae-409c4d60570f)
![395062253_1568958183639010_3008104083537784579_n](https://github.com/kachurovskiy/nanoels/assets/2408274/e781c582-ac7c-4596-91d3-91cc453b75bd)
![393969123_267569872405413_7326707628221027753_n](https://github.com/kachurovskiy/nanoels/assets/2408274/8e347e84-4eb2-48ce-a5e1-ac09d8f60d9e)

**Reply by kachurovskiy** [reply]
You could use the available belt seat but this usually leads to fractional impulses per revolution which is hard to deal with.

Could you make a small gear train where spindle gear transfers rotation to a free-spinning gear that is coaxial with another one that transfers to the encoder?

![PXL_20231022_155122774](https://github.com/kachurovskiy/nanoels/assets/517919/5ce23269-ab13-4a07-93ee-467d5a4a63ef)

**Reply by Festivejelly** [reply]
Ah yes this could work. I could mount some sort of rod on a bearing thats embedded into some sort of base plate. Ill do some cad and test prints. Great idea thanks.

**Reply by AlanMerry** [reply]
I have a Opti Turn TU2004V lathe which is almost exactly the same as your WM180.

I am in the process of fitting the Nano H4.

For the encoder, I actually have purchased three, but only one will fit along with the 80tooth gear. There are subtle differences in the height of these units but this one looks like it is a bit lower.

This is the encoder I will most likely use, the 1000 pulse version which will be halved due to the 40/80 gear ratio:-
https://www.aliexpress.com/item/1005005780216914.html?spm=a2g0o.store_pc_groupList.8148356.6.120575eadJ7oC9&pdp_npi=4%40dis%21NZD%21NZ%24%2024.66%21NZ%24%2012.32%21%21%2113.99%216.99%21%40210318ef16983066243813322ebcb8%2112000034527971539%21sh%21NZ%212223447251%21

I have also printed this 80 tooth gear which I modified to allow for the 6mm shaft with the flat section.

[80t_Encoder_Gear_6mm_hole.zip](https://github.com/kachurovskiy/nanoels/files/13174645/80t_Encoder_Gear_6mm_hole.zip)

Waiting for all of the motors etc to arrive so early days.

**Reply by Festivejelly** [reply]
Thanks Alan.

They do indeed look similar. I wish sellers would include the dimensions for these encoders. I'll have a shop around and see if I can find a shorter one. As Maxim suggested I could make a small coaxial shaft with a gear that will transfer the motion to the encoder. Im in the process of drawing up the cad designs for it. Ill post it on here when its done.

**Comment by AlanMerry** [comment]
Hi,

The encoder link I provided does, have the dimensions. It should fit.
Kind regards



Alan Merry

Director

Kiwiboards Limited

34 Bay Road
St Heliers
Auckland
New Zealand

e: ***@***.******@***.***

m: +64 204 1846245

w: http://www.kiwiboards.co.nz

27/10/2023 3:22:49 am Nicholas John ***@***.***>:

**Reply by Festivejelly** [reply]
I still think its a bit big for my lathe sadly. Looking at the diagram it seems that the base height of the encoder is 31mm. But the distance between the body of my lathe and the gear on the spindle is only 25mm, so I think ill still need to make some tweaks to my setup.

**Comment by Festivejelly** [comment]
So far this is what ive come up with

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/4a10f7d6-30bf-405f-bc22-59e279adc4c2)

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/a048733e-d4cf-416f-8347-e810287a88ea)
 
 The dotted outline is the rough outline profile of the lathe. So it should fit in. Though I do worry that 3d printing so many gears could cause inaccuracies.
 
 The rod is supported by a 608zz bearing. Ill need to come up with a creative way to mount all this though.

**Comment by Festivejelly** [comment]
Made a mount. Probably need a better way of constraining the 8mm shaft though so it cant move up or down.

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/3c6b86f4-57cc-4d7f-b2f6-4d3e50a388e2)

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/6acc5d2e-f6a7-4442-a580-f93c4bdd3f28)

**Comment by Festivejelly** [comment]
Refined it a bit more. I will test it out but if anyone wants the files i'll be uploading them to thingiverse after I verify it works. Its fairly complicated. The gear in the middle is a double gear. Its held in place with a grub screw. As is the gear on the encoder.

An 8mm shaft supports the central gears, each end has a 608zz bearing which needs to be pressed into place on the 3d printed top and bottom

I have a solid plate behind where I plan to mount it. So the plan for me is to drill holes into it and mount it with 4 x m4 bolts.

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/74e99f52-cbc6-4e5d-89b5-7f2874c8f035)
![image](https://github.com/kachurovskiy/nanoels/assets/2408274/0e2c2685-a85a-4975-9fbc-9bb94e119153)

**Reply by kachurovskiy** [reply]
Looks very nice!

**Comment by Festivejelly** [comment]
The next question is... how the heck do I figure out what ratio I need to set the encoder on. Math has never been my strong point. I know that the gears on the spindle is 40 teeth this meshes with a 50 tooth gear which has a 41 tooth gear below it which in turn meshes with a 60 tooth gear on the encoder.

Here is what chatGPT said, but it could literally say anything and I wouldnt know if it was right :D

Compute Each Pair's Gear Ratio:
For each pair of gears:

**Gear Ratio** = Number of Teeth on Driven Gear / Number of Teeth on Driving Gear

a. For the first pair (lathe spindle gear and its meshing gear):

**Ratio 1** = 50 / 40

b. For the second pair (41-tooth gear and encoder gear):

**Ratio 2** = 60 / 41

Compute Overall Gear Ratio:
The overall gear ratio is the product of the individual ratios:

**Overall Ratio** = Ratio 1 × Ratio 2

Plugging in the values:

Overall Ratio = 50 / 40 × 60 / 41

Interpret the Ratio:
Let's compute the overall ratio using the provided values:

**Overall Ratio** = 1.25 × 1.4634 ≈ 1.8304

This means that for every 1.8304 rotations of the encoder, the lathe spindle will complete 1 rotation.

**Reply by kachurovskiy** [reply]
How many teeth are on each gear? Assuming it's 40-60-41-60 starting from the spindle, your pulses per revioution `PPR` would be `EPPR * 40 / 60 * 41 / 60` where EPPR is the number of pulses your encoder has per revolution. You would want to adjust the number of teeth on your gears so that the expression results in a non-fractional number.

**Reply by Festivejelly** [reply]
So I think if I use 40(spindle gear)-50-45-72 that gives a ratio of 2:1 so then I could just double that in the config? My encoder has 600 pulses per revolution so I guess that makes that 1200

**Reply by kachurovskiy** [reply]
40/50*45/72 is 0.5 so it would be 300 pulses on encoder for each spindle turn which is okay but somewhat on the low side.

**Comment by Festivejelly** [comment]
Okay, I might have a design that finally works

Its a 40 - 50 - 40 which should give 1:1

![image](https://github.com/kachurovskiy/nanoels/assets/2408274/0bcd4745-58d2-4e49-9cd8-1b3f576db7f4)

It can either be bolted straight through the legs into drilled holes in the frame, or you can just put heat inserts in the base plate and screw the top into that. Then affix with sticky take to the body of the lathe. I'll likely try it with tape to get positioning, then bolt it later when im happy with position.

This was quite a challenge! Not a lot of room in my lathe!
